opentelemetry-otlp/src/exporter/http/logs.rs

```diff
@@ -28,7 +28,7 @@ impl LogExporter for OtlpHttpClient {
                 .method(Method::POST)
                 .uri(&self.collector_endpoint)
                 .header(CONTENT_TYPE, content_type)
-                .body(body)
+                .body(body.into())
```

Comment:
I believe this simply takes ownership of the Vec's buffer and wraps it in a Bytes object? Just want to ensure there is no copy operation here.

Reply:
As I see from source code the only allocation possible is new Box allocated in case of buffer length != capacity: https://docs.rs/bytes/latest/src/bytes/bytes.rs.html#964-996. The ownership of buffer is just transferred.